keyboards/handwired/horrortroll/k552/keymaps/custom/oled/oled.c

```diff
@@ -0,0 +1,75 @@
+#include <stdio.h>
+#include "layers.c"
+
+// OLED animation
+#include "logo.c"
+#include "bongocat.c"
+#include "wave.c"
+#include "galaxy.c"
+
+char wpm_str[10];
+
+#ifdef OLED_ENABLE
+uint16_t startup_timer; 
+
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+    startup_timer = timer_read();
+    wave_timer = timer_read();
+
+    return rotation;
+}
+
+void oled_task_user(void) {
```

Comment:
`oled_task_user()`'s signature has changed; it needs to `return` either `false` or `true`.

```suggestion
bool oled_task_user(void) {
```